lib/OpenQA.pm

```diff
@@ -18,69 +18,5 @@ package OpenQA;
 use strict;
 use warnings;
```

Comment:
why not delete the whole file?

Reply:
Since openQA is structured as a normal Perl module distribution, it makes sense to keep a customary `DistroName.pm` file. It usually holds at least a `$VERSION` variable and some documentation, which might make sense for us to add.